test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py

```diff
@@ -188,6 +188,49 @@ def test_load_two_models():
     assert len(res3) == 2
 
 
+@pytest.mark.model("half_plus_three, half_plus_two")
+@pytest.mark.processor("cpu")
```

Comment:
nit: is this explicitly a MME test? can you add @pytest.mark.integration("mme") to this as well if so?

Reply:
will do